test/generator.test.js

```diff
@@ -329,6 +331,26 @@ describe('Generator', () => {
       expect(utils.fetchSpec.mock.calls[0][0]).toBe(asyncapiURL);
       expect(generateMock.mock.calls[0][0]).toBe('fake text');
     });
+
+    it('works with a path to registry', async () => {
+      log.debug = jest.fn();
+      const gen = new Generator('nameOfTestTemplate', __dirname, {debug: true, registry: {url: 'some.registry.com', username: 'user', password: 'password', token: 'token'}});
+      await gen.installTemplate();
+      setTimeout(() => { // This puts the call at the end of the Node.js event loop queue.
+        expect(arboristMock.reify).toHaveBeenCalledTimes(1);
```

Comment:
you need to define a new mock as normally you would not have to do timeouts

Reply:
Remove value of timeout @derberg